### DISCUSSION TITLE
Can't open github links in github app instead it opens in chrome and other search engines

### DISCUSSION THREAD

**Discussion by GayathiriRaghunathan** [community/community#3281] [Mobile] [answered]
How to open links in github

**Comment by candyho** [comment, marked as answer]
Hey there, currently the app only support to open a specific set of links in-app, for those that we don't support we'll redirect to the browser. Are you using an Android device? If so, I think you might have selected Chrome (or other browsers) to open all github links. 

The first time you tap on a GitHub link, the OS will show you two options - open in-app or in an external browser. There should be a button for you to select “Only this time” or “Always” - you might have choose "Always" for the first time. Would recommend you to reset your Android default apps, here's a handy link - https://www.techrepublic.com/article/pro-tip-resetting-android-default-apps/

Hope this helps.

**Reply by TiansuYu** [reply]
For me, it simply redirects on README page for a repo. I am not sure if this is intended behaviour or simply bug by now.

**Reply by Darkhost1999** [reply]
I am experiencing an issue with the GitHub bot on Discord. When I click on hyperlinks, they open in Firefox. However, if I copy and paste the same link into my notes or a chat, it opens correctly in the GitHub app. This issue only occurs when I click on links directly from the GitHub bot, regardless of the web browser I use.

**Reply by IArnaut2** [reply]
I CAN'T CREATE ISSUES INSIDE GITHUB APP!!! I GET REDIRECTED TO THE BROWSER!!!
